ide/lsp.client/src/org/netbeans/modules/lsp/client/LSPBindings.java

```diff
@@ -441,11 +471,14 @@ public static synchronized Set<LSPBindings> getAllBindings() {
     private final LanguageServer server;
     private final InitializeResult initResult;
     private final Process process;
+    // LSP Server specific request processor
+    public final RequestProcessor worker;
 
     private LSPBindings(LanguageServer server, InitializeResult initResult, Process process) {
         this.server = server;
         this.initResult = initResult;
         this.process = process;
+        this.worker = new RequestProcessor(server.getClass().getName() + "WORKER", 1, false, false); // NOI18N
```

Comment:
```suggestion
        this.worker = new RequestProcessor(server.getClass().getName() + ".WORKER", 1, false, false); // NOI18N
```

See comment in line 113.